cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -558,6 +558,36 @@ storage:
             volumeMounts:
             - name: config-volume
               mountPath: /etc/nginx
+          - name: skipper-proxy
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.10.233
+            args:
+            - skipper
+            - -address=:8443
+            - -tls-cert=/etc/kubernetes/ssl/apiserver.pem
+            - -tls-key=/etc/kubernetes/ssl/apiserver-key.pem
+            - -insecure
+            - -inline-routes
+            - 'z: JWTPayloadAnyKV("iss", "https://identity.zalando.com") -> disableAccessLog() -> "https://127.0.0.1:443"; all: * -> enableAccessLog() -> "https://127.0.0.1:443";'
+            ports:
+            - containerPort: 8443
+            readinessProbe:
+              httpGet:
+                path: /metrics
+                port: 9911
+              initialDelaySeconds: 60
```

Comment:
That's quite a lot. Why a minute?

Reply:
Taken from [how we run skipper-ingress](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/skipper/deployment.yaml#L114). But I'm open for suggestions.

Reply:
I would think it's even too much for skipper ingress, but there we need skipper to connect to the apiserver and configure all the routes. Here we don't need that, so should be much faster.

Reply:
The 60 seconds in skipper-ingress is a hack to slow down the rotation of Skippers during upgrade IIRC. The readiness probe doesn't even do much here, since the ELB will contact Skipper anyway, so we could safely reduce it.

Reply:
Skipper starts in `<5` seconds if you load from routes file in inline routes. You do not need to set initialDelaySeconds.

Reply:
set to 10.